src/pages/search-help.vue

```diff
@@ -202,17 +222,29 @@
   </VContentPage>
 </template>
 
-<script>
+<script lang="ts">
+import { defineComponent, useContext } from '@nuxtjs/composition-api'
+
 import VLink from '~/components/VLink.vue'
 import VContentPage from '~/components/VContentPage.vue'
 
-const SearchHelpPage = {
-  name: 'search-help-page',
+const SearchHelpPage = defineComponent({
+  name: 'SearchHelpPage',
```

Comment:
Missing a `V`? I think we haven't used this convention for pages (not sure if it even matters 🤷  since they're never used as components).
```suggestion
  name: 'VSearchHelpPage',
```

Reply:
Thank you, great catch! And I think we did not use this convention for pages, right. 🚴‍♀️ 🛖 😆